database/migrations/2024_03_23_071253_create_inquiry_status_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('inquiry_status', function (Blueprint $table) {
+            $table->id();
+            $table->string('status')->unique()->comment('1:unsolved, 2:answer, 3:solved');
```

Comment:
use int for status

Reply:
We use id for primarykey and the content of id in status, so we think status should be string , but rather fix to 
`string('status' , 15)`

Reply:
Sorry this is a kind of master table , so this one must be not changable. I will modify this again.